core/experiments/manager.php

```diff
@@ -155,6 +155,19 @@ public function add_feature( array $options ) {
 		return $experimental_data;
 	}
 
+	private function install_compare( $version ) {
+		$installs_history = Upgrade_Manager::get_installs_history();
+
+		end( $installs_history );
+		$cleaned_version = preg_replace( '/-(beta|cloud|dev)\d*$/', '', key( $installs_history ) );
+
+		return version_compare(
+			$cleaned_version,
+			$version,
+			'>='
+		);
+	}
```

Comment:
why don't you fix it in the upgrade manager?

Reply:
Can't change it...